main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -309,14 +310,14 @@ private void loadFeedInfo(GtfsTableContainer<GtfsFeedInfo> feedTable) {
     feedInfo.put(
         FEED_INFO_FEED_LANGUAGE, info == null ? "N/A" : info.feedLang().getDisplayLanguage());
     if (feedTable.hasColumn(GtfsFeedInfo.FEED_START_DATE_FIELD_NAME)) {
-      feedInfo.put(
-          FEED_INFO_FEED_START_DATE,
-          info == null ? "N/A" : info.feedStartDate().getLocalDate().toString());
+      LocalDate localDate = info.feedStartDate().getLocalDate();
+      String displayDate = localDate.equals(LocalDate.of(1970, 1, 1)) ? "N/A" : localDate.toString();
+      feedInfo.put(FEED_INFO_FEED_START_DATE, info == null ? "N/A" : displayDate);
```

Comment:
The `N/A` should be rendered in the HTML file. Having the `N\A` in the feedInfo instance will replicate the value in the JSON file. This leads to issues on the consumer side when parsing a valid date. We have an example in the email of an invalid value being put in the JSON file, an example of a JSON report. https://gtfs-validator-results.mobilitydata.org/4a1ab20a-5e9e-47fb-8b86-45bc60e88b3f/report.json